lightrag/lightrag.py

```
@@ -305,33 +307,115 @@ async def _insert_done(self):
+                        + dp["src_id"]
+                        + dp["tgt_id"]
+                        + dp["description"],
+                    }
+                    for dp in all_relationships_data
+                }
+                await self.relationships_vdb.upsert(data_for_vdb)
+        finally:
+            if update_storage:
+                await self._insert_done()
+
     def query(self, query: str, param: QueryParam = QueryParam()):
```

Ruff (0.8.0)

Do not perform function call `QueryParam` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)